### ISSUE TITLE
posting kernel binaries for releases

### ISSUE THREAD

**Issue by earney** [LibertyOS-Development/kernel#7]
would it be possible to post kernel binaries for releases?  That way someone does not have to compile the source code to try it out.

**Comment by danielteberian** [comment]
Yes. I might find somebody to be in charge of managing binary releases, and building binaries for different systems. I can do this today/tomorrow. :)

**Comment by xonmello** [comment]
Taking a look at this, should be able to automate this with [github actions](https://github.com/features/actions), I can take a look at it tomorrow and see if I can get that setup for us

**Comment by danielteberian** [comment]
> Taking a look at this, should be able to automate this with [github actions](https://github.com/features/actions), I can take a look at it tomorrow and see if I can get that setup for us 

Thank you. By the way, "gh release" might work.
